### PR TITLE
truncate select task text to prevent overlap with the dropdown icon

### PR DESCRIPTION
**Pull request by debsmita1**
**Fixes:**
https://issues.redhat.com/browse/ODC-5558

**Root analysis:**
Because of localization, the text overlaps with the dropdown icon.

**Solution description:**
- added  ellipsis to localized `Select Task` text
- show tooltip only when an ellipsis is present

**GIF:**
selecttask

**Browser conformance:**
- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by debsmita1** [comment]
@bgliwa01

**Comment by andrewballantyne** [comment]
FWIW, this was brought up at the Pipelines UX meeting and @bgliwa01 will be reaching out to the Admin team to inquire if this is a unique situation or if they have had to address this in other spots.

**Comment by jerolimov** [comment]
For the tooltip discussion. This tooltip was shown in the `Pipeline` and `PipelineRun` visualization:

image

image

If we add a tooltip here, it would makes sense to add it for the selected tasks which are also truncated: 

image

The problem is that we should keep in mind the + buttons here.

**Review by jerolimov — APPROVED** [review]
From my perspective this code change is small and better then before. I approve but are open for feedback from @bgliwa01 . Hope this is okay for everyone. :)

/approve

**Comment by jerolimov** [comment]
Reviewed the new approach tooltip again, works fine. :+1:

Keep my /approve

**Comment by divyanshiGupta** [comment]
Thanks @debsmita1 for making the changes. Code looks good to me. 
I will add lgtm once @bgliwa01 gives her approval for the new UX.

**Comment by debsmita1** [comment]
rebased

**Review by bgliwa01 — APPROVED** [review]
lgtm

**Comment by jerolimov** [comment]
> I will add lgtm once @bgliwa01 gives her approval for the new UX.

As Divyanshi and Brigid accepted this.. thanks both :)

/lgtm

**Pull request merged**
